pkg/test/framework/components/sleep/kube.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sleep
+
+import (
+	"io/ioutil"
+	"path"
+
+	"istio.io/istio/pkg/test/env"
+	"istio.io/istio/pkg/test/framework/components/deployment"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"istio.io/istio/pkg/test/framework/resource"
+)
+
+type sleepConfig string
+
+type sleepComponent struct {
+	deployment.Instance
+}
+
+const (
+	// Sleep uses "sleep.yaml"
+	Sleep          sleepConfig = "sleep.yaml"
+)
+
+func deploy(ctx resource.Context, cfg Config) (i SleepInstance, err error) {
+	ns := cfg.Namespace
+	if ns == nil {
+		ns, err = namespace.Claim(ctx, "default")
```

Comment:
I don't think, it's suggested to deploy in default namespace as cleanup becomes tough

Reply:
Removed this component